ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -202,124 +333,22 @@ export const AccountListMenu = ({ onClose }) => {
-                    if (getEnvironmentType() === ENVIRONMENT_TYPE_POPUP) {
-                      global.platform.openExtensionInBrowser(
-                        CONNECT_HARDWARE_ROUTE,
-                      );
-                    } else {
-                      history.push(CONNECT_HARDWARE_ROUTE);
-                    }
-                  }}
-                >
-                  {t('addHardwareWallet')}
-                </ButtonLink>
-              </Box>
-              {
-                ///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
-                addSnapAccountEnabled && (
-                  <Box marginTop={4}>
-                    <ButtonLink
-                      size={Size.SM}
-                      startIconName={IconName.Snaps}
-                      onClick={() => {
-                        dispatch(toggleAccountMenu());
-                        getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
-                          ? global.platform.openExtensionInBrowser(
-                              ADD_SNAP_ACCOUNT_ROUTE,
-                              null,
-                              true,
-                            )
-                          : history.push(ADD_SNAP_ACCOUNT_ROUTE);
-                      }}
-                    >
-                      {t('settingAddSnapAccount')}
-                    </ButtonLink>
-                  </Box>
-                )
-                ///: END:ONLY_INCLUDE_IN
-              }
-              {
-                ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
-                <Box marginTop={4}>
-                  <ButtonLink
-                    size={Size.SM}
-                    startIconName={IconName.Custody}
-                    onClick={() => {
-                      dispatch(toggleAccountMenu());
-                      trackEvent({
-                        category: MetaMetricsEventCategory.Navigation,
-                        event:
-                          MetaMetricsEventName.ConnectCustodialAccountClicked,
-                      });
-                      if (getEnvironmentType() === ENVIRONMENT_TYPE_POPUP) {
-                        global.platform.openExtensionInBrowser(
-                          CUSTODY_ACCOUNT_ROUTE,
-                        );
-                      } else {
-                        history.push(CUSTODY_ACCOUNT_ROUTE);
-                      }
-                    }}
-                  >
-                    {t('connectCustodialAccountMenu')}
-                  </ButtonLink>
-                </Box>
-                ///: END:ONLY_INCLUDE_IN
-              }
+            {/* Add / Import / Hardware button */}
+            <Box
+              paddingTop={4}
+              paddingBottom={4}
+              alignItems={AlignItems.center}
+              display={Display.Flex}
+            >
+              <ButtonSecondary
+                startIconName={IconName.Add}
+                variant={ButtonVariant.Secondary}
+                style={{ margin: '0 auto' }}
```

Comment:
I think we can remove the inline styling here, it should have a paddingLeft and right 16px same as the cell children. Also, the CTA size should be LG
```suggestion
            <Box
              paddingTop={2}
              paddingBottom={4}
              paddingLeft={4}
              paddingRight={4}
              alignItems={AlignItems.center}
              display={Display.Flex}
            >
              <ButtonSecondary
                startIconName={IconName.Add}
                variant={ButtonVariant.Secondary}
                size={ButtonSecondarySize.Lg}
                block
```

Reply:
Implemented!